store/mongo/migration_0_2_0.go

```diff
@@ -30,6 +32,31 @@ func (m *migration_0_2_0) Up(from migrate.Version) error {
 	s := m.ms.session.Copy()
 	defer s.Close()
 
+	// get all attribute names
+	names, err := m.ms.GetAllAttributeNames(m.ctx)
+	if err != nil {
+		return errors.Wrap(err, "failed to get attribute names")
+	}
+
+	// rename every attribute occurrence to scoped version, add scope field
+	// hacky - we're doing it in two runs, but dead simple
+	coll := s.DB(mstore.DbFromContext(m.ctx, DbName)).C(DbDevicesColl)
+	for _, n := range names {
+		nold := fmt.Sprintf("%s.%s", DbDevAttributes, n)
+		nnew := fmt.Sprintf("%s.%s_%s", DbDevAttributes, DbScopeInventory, n)
```

Comment:
in the store, we are using "-", not "_"
https://github.com/mendersoftware/inventory/blob/db8e5d0dae65de072c5ef264af924e1889f41a47/store/mongo/datastore_mongo.go#L350

Reply:
fixed - also my test wasn't testing anything, so fixed that too (along with the implementation)